sycl/doc/design/OffloadDesign.md

```diff
@@ -296,6 +296,125 @@ resemble `--gpu-tool-arg=<arch> <arg>`.  This corresponds to the existing
 option syntax of `-fsycl-targets=intel_gpu_arch` where `arch` can be a fixed
 set of targets.
 
+#### --offload-arch
+
+For SYCL offloading to Intel GPUs, Intel CPUs, NVidia and AMD GPUs, specify the device architecture using ``--offload-arch`` option. For instance 
+  ``--offload-arch=sm_80`` to target an NVidia Tesla A100, 
+  ``--offload-arch=gfx90a`` to target an AMD Instinct MI250X, or 
+  ``--offload-arch=sm_80,gfx90a`` to target both.
+For Intel Graphics AOT target, valid values for ``--offload-arch`` are the
+valid device names accepted by OCLOC (the Intel GPU AOT compiler) via the ``-device`` option.
```

Comment:
Is this last sentence correct?  The table below has separate columns for "``--offload-arch`` accepted value" and "OCLOC -device value".  If the two values are always the same, why does the table have two columns?  Looking at the table, it seems like the values are not always the same between ``--offload-arch`` and the OCLOC `-device` flag.

Reply:
> For Intel Graphics AOT target, valid values for ``--offload-arch`` are the
> valid device names accepted by OCLOC (the Intel GPU AOT compiler) via the ``-device`` option.


You are correct. The two values are not always the same. 
A group of valid `--offload-arch` values are sometimes mapped to a single `OCLOC -device` value.
Does the sentence below sound right?

```
For Intel Graphics AOT target, valid values for ``--offload-arch`` are mapped to 
valid device names accepted by OCLOC (the Intel GPU AOT compiler) via the ``-device`` option.
```

Reply:
Yes, that looks good!